pkg/database/plugin.go

```diff
@@ -185,6 +188,9 @@ type iOffsetCollection interface {
 }
 
 type iPinCollection interface {
+	// InsertPins - Inserts a list of pins - fails if they already exist, so caller can fall back to UpsertPin individually
```

Comment:
> fails if they already exist, so caller can fall back to UpsertPin individually

This is a useful bit of information that (I think) is also relevant on some of the other multi-insert calls added recently.

Reply:
It wasn't before, but it is now after this PR 👍